vividus/src/integrationTest/resources/properties/configuration.properties

```diff
@@ -1,2 +1,4 @@
 profile-to-use=unknown
 environments-to-use=basicenv
+
+bdd.configuration.formats=JSON,XML
```

Comment:
it's better to keep such properties in `suite` properties group (also `environment` group is acceptable), but not in `configuration.properties`